awesomeConfig.cmake

```diff
@@ -332,6 +334,7 @@ if(DO_COVERAGE)
                     ${BUILD_DIR}/${file}
                     COPYONLY)
     endforeach()
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -O0 --coverage -fprofile-arcs -ftest-coverage")
```

Comment:
LDFLAGS does not seem to be necessary (as the coverage report shows).
Should I re-add it here?